File: mentor-on-demand-system-v3/src/main/java/com/ibm/mods/training/model/Training.java
package com.ibm.mods.training.model;

import java.sql.Time;
import java.util.Date;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;
import javax.persistence.Temporal;
import javax.persistence.TemporalType;

import com.fasterxml.jackson.annotation.JsonFormat;

@Entity
@Table(name = "trainings")
public class Training {

	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private long id;
	
	@Column(name = "user_id")
	private long userId;
	
	@Column(name = "mentor_id")
	private long mentorId;
	
	@Column(name = "skill_id")
	private long skillId;
	
	@Column(name = "status")
	private String status;
	
	@Column(name = "progress")
	private int progress;
	
	@Column(name = "rating")
	private int rating;
	
	@Temporal(TemporalType.DATE)
	@Column(name = "start_date")
	private Date startDate;
	
	@Temporal(TemporalType.DATE)
	@Column(name = "end_date")
	private Date endDate;
	
	@JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "HH:mm")
	@Temporal(TemporalType.TIME)
	@Column(name = "start_time")
	private Date startTime;
	
	@JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "HH:mm")
	@Temporal(TemporalType.TIME)
	@Column(name = "end_time")
	private Date endTime;
	
	public Training() {
		
	}
	
	public long getId() {
		return id;
	}
	
	public void setId(long id) {
		this.id = id;
	}
	
	public long getUserId() {
		return userId;
	}
	
	public void setUserId(long userId) {
		this.userId = userId;
	}
	
	public long getMentorId() {
		return mentorId;
	}
	
	public void setMentorId(long mentorId) {
		this.mentorId = mentorId;
	}
	
	public long getSkillId() {
		return skillId;
	}
	
	public void setSkillId(long skillId) {
		this.skillId = skillId;
	}
	
	public String getStatus() {
		return status;
	}
	
	public void setStatus(String status) {
		this.status = status;
	}
	
	public int getProgress() {
		return progress;
	}
	
	public void setProgress(int progress) {
		this.progress = progress;
	}
	
	public int getRating() {
		return rating;
	}
	
	public void setRating(int rating) {
		this.rating = rating;
	}
	
	public Date getStartDate() {
		return startDate;
	}
	
	public void setStartDate(Date startDate) {
		this.startDate = startDate;
	}
	
	public Date getEndDate() {
		return endDate;
	}
	
	public void setEndDate(Date endDate) {
		this.endDate = endDate;
	}
	
	public Date getStartTime() {
		return startTime;
	}
	
	public void setStartTime(Date startTime) {
		this.startTime = startTime;
	}
	
	public Date getEndTime() {
		return endTime;
	}
	
	public void setEndTime(Date endTime) {
		this.endTime = endTime;
	}
	
}

File: mentor-on-demand-app-client/src/app/models/training.ts
import { Time } from '@angular/common';
import { typeWithParameters } from '@angular/compiler/src/render3/util';

export class Training {
    id: number;
    userId: number;
    mentorId: number;
    mentorSkillId: number;
    skillId: number;
    status: string;
    progress: number;
    rating: number;
    startDate: string;
    endDate: string;
    startTime: string;
    endTime: string;
    daysOfSession: string;
    trainingTotalHours: number;
    trainingTotalAmount: number;

    constructor(id: number,
                userId: number,
                mentorId: number,
                mentorSkillId: number,
                skillId: number,
                status: string,
                progress: number,
                rating: number,
                startDate: string,
                endDate: string,
                startTime: string,
                endTime: string,
                daysOfSession: string,
                trainingTotalHours: number,
                trainingTotalAmount: number) {
        this.id = id;
        this.userId = userId;
        this.mentorId = mentorId;
        this.mentorSkillId = mentorSkillId;
        this.skillId = skillId;
        this.status = status;
        this.progress = progress;
        this.rating = rating;
        this.startDate = startDate;
        this.endDate = endDate;
        this.startTime = startTime;
        this.endTime = endTime;
        this.daysOfSession = daysOfSession;
        this.trainingTotalHours = trainingTotalHours;
        this.trainingTotalAmount = trainingTotalAmount;
    }

}